docs/getting-started/powerstrip/EnvironmentSetup.md

```diff
@@ -70,10 +70,10 @@ If you prefer not to do this you can still run the demo but remember to run
 ### Getting calicoctl Binary
 
 Get the calicoctl binary onto each host.  Don't use the latest 
-[release](https://github.com/Metaswitch/calico-docker/releases/) from github.
-Version v0.4.8 is required.
+[release](https://github.com/projectcalico/calico-docker/releases/) from github.
+Version v0.4.9 is required.
 
-	wget https://github.com/Metaswitch/calico-docker/releases/download/v0.4.8/calicoctl
+	wget https://github.com/projectcalico/calico-docker/releases/download/v0.4.9/calicoctl
```

Comment:
Did we actually test these against 0.4.9 ... I'm just surprised we didn't pick this up as part of the docs rework and testing.